app/audit/DataEventFactory.scala

```diff
@@ -166,4 +171,41 @@ object DataEventFactory {
     hc.toAuditTags(transactionName, request.path) ++
       Map(hc.names.deviceID -> hc.deviceID.getOrElse("-"))
   }
+
+  private def incomeToJson(income: Option[Income]): JsObject = {
+    income match {
+      case Some(income) =>
+
+        def mapKeyName(key: String) = key match {
+            case "monthlyIncome" => "monthlyIncomeAfterTax"
+            case "otherIncome"   => "otherMonthlyIncome"
+            case other           => other
+          }
+
+        val map = income.budgetLines.map { budgetLine =>
+          mapKeyName(budgetLine.category.entryName) -> JsString(formatToCurrencyString(budgetLine.amount))
+        }.toMap
+        JsObject(map) + ("totalIncome" -> JsString(formatToCurrencyString(income.totalIncome)))
```

Comment:
whoops, I said `formatToCurrencyString` would do what we wanted but I forgot what it actually did - we probably need our own format method in this class cos `formatToCurrencyString` will format say `500` to `"500"` instead of `"500.00"`. There are also some other money amounts in the audit event which should be formatted this way. We need two decimal places always in the audit event from what the CIP team said.

Reply:
Agreed to change the implementation to using case classes and format all currency inputs consistently to Sting with trailing zeros